src/aui/tabart.cpp

```diff
@@ -1522,6 +1531,10 @@ wxSize wxAuiSimpleTabArt::GetTabSize(wxDC& dc, wxWindow* WXUNUSED(wnd), const wx
     wxCoord tabHeight = measuredTextY + 4;
     wxCoord tabWidth = measuredTextX + tabHeight + 5;
 
+    if (bitmap.IsOk()) {
+        tabWidth += bitmap.GetWidth() + 6;
```

Comment:
The +6 border here and the others above, should we make those member variables which can be set, this might reduce the amount of code that someone would have to write just to change the tabs border.